src/lib/default_english_messages.js

```diff
@@ -38,7 +38,13 @@ var defaultEnglishMessages = {
     "gutter-tooltip.aria-label.warning.singular": "warning",
     "gutter-tooltip.aria-label.warning.plural": "warnings",
     "gutter-tooltip.aria-label.info.singular": "information message",
-    "gutter-tooltip.aria-label.info.plural": "information messages"
+    "gutter-tooltip.aria-label.info.plural": "information messages",
+    "gutter.annotation.aria-label.security": "Read annotations row $0",
```

Comment:
Like we have here for `error`, `warning`, `info`, these strings should include the severity of the annotation (so they should be something like `Security finding, read annotations row $0`). Probably best to change the string in `gutter.js`.